src/diagnostics/defaults/moisture_model.jl

```diff
@@ -0,0 +1,22 @@
+# FIXME: Gabriele added this as an example. Put something meaningful here!
+function get_default_diagnostics(::T) where {T <: DryModel}
+    return vcat(
+        get_daily_average("air_density"),
```

Comment:
I'm not sure about the variadic option. Part of me thinks it would be better to return a single value, and require users to broadcast, e.g.
```julia
        get_daily_average.(["air_density", "air_pressure"])
```